src/components/structures/BottomLeftMenu.js

```diff
@@ -131,19 +132,19 @@ module.exports = React.createClass({
                     { homeButton }
                     <AccessibleButton className="mx_BottomLeftMenu_people" onClick={ this.onPeopleClick } onMouseEnter={ this.onPeopleMouseEnter } onMouseLeave={ this.onPeopleMouseLeave } >
                         <TintableSvg src="img/icons-people.svg" width="25" height="25" />
-                        { this.getLabel("Start chat", this.state.peopleHover) }
+                        { this.getLabel(counterpart.translate('Start chat'), this.state.peopleHover) }
```

Comment:
In my opinion this should be much shorter, `counterpart.translate(....` is a little verbose, might as well have `translate` on its own
(I think in this case `counterpart()` would work, so you could just import it into a const named `translate`)